lib/active_merchant/billing/gateways/worldpay.rb

```diff
@@ -698,7 +699,7 @@ def add_stored_credential_using_normalized_fields(xml, options)
         stored_credential_params = generate_stored_credential_params(is_initial_transaction, reason)
 
         xml.storedCredentials stored_credential_params do
-          xml.schemeTransactionIdentifier options[:stored_credential][:network_transaction_id] if options[:stored_credential][:network_transaction_id] && !is_initial_transaction
+          xml.schemeTransactionIdentifier network_transaction_id(options) if network_transaction_id(options) && !is_initial_transaction
```

Comment:
Considering this method is for fields in stored_credentials I think it's best to keep as it was before. There could be instance where the customer passed stored_credential_transaction_id and also has stored_credential.

Reply:
So I updated this because if a merchant passes in a GSF for NTID it should take priority over the one we have on file. That's how we handle it on other gateways.

Reply:
Are you worried this would be a breaking change?

Reply:
That's what also came to mind but I noticed we don't do that in WorldPay for Core. I think it's better to change it core to keep uniformity and I also noticed that other stored credentials GSF are available but we aren't updating those like stored_credential_usage and stored_credential_initiated_reason

Reply:
In core? I don't think we should hide that logic somewhere else. I didn't add the other fields because if you're using the stored credential framework you wouldn't want to override the mappings but you may want to override the NTID if you brought one from somewhere else though I'm second guessing that now. 

Stripe PI does this in AM by first setting the `stored_credential` value then overriding with the `options` value if present. fwiw it seems we do this already for TPV on worldpay (not sure why TPV has ntids 🤷🏽 )

Reply:
I removed this and just left the stored credential for NT change